src/app/components/Byline/index.tsx

```diff
@@ -21,154 +22,172 @@ const Byline = ({
 
   const bylineValues = bylineExtractor(blocks);
 
-  if (!bylineValues) return null;
+  const contributors = bylineValues.map(values => {
+    if (!values) return null;
+    const {
+      authorName,
+      jobRole,
+      twitterText,
+      twitterLink,
+      authorImage,
+      location,
+      authorTopicUrl,
+    } = values;
 
-  const {
-    authorName,
-    jobRole,
-    twitterText,
-    twitterLink,
-    authorImage,
-    location,
-    authorTopicUrl,
-  } = bylineValues;
+    const authorTranslated = pathOr(
+      'Author',
+      ['byline', 'author'],
+      translations,
+    );
+    const articleInformationTranslated = pathOr(
+      'Article information',
+      ['byline', 'articleInformation'],
+      translations,
+    );
+    const jobRoleTranslated = pathOr('Role', ['byline', 'role'], translations);
 
-  const authorTranslated = pathOr('Author', ['byline', 'author'], translations);
-  const articleInformationTranslated = pathOr(
-    'Article information',
-    ['byline', 'articleInformation'],
-    translations,
-  );
-  const jobRoleTranslated = pathOr('Role', ['byline', 'role'], translations);
-
-  const reportingFromTranslated = pathOr(
-    'Reporting from',
-    ['byline', 'reportingFrom'],
-    translations,
-  );
-  return (
-    <section
-      css={BylineCss.bylineSection}
-      role="region"
-      aria-labelledby="article-byline"
-    >
-      <VisuallyHiddenText as="strong" id="article-byline" aria-hidden>
-        {articleInformationTranslated}
-      </VisuallyHiddenText>
-      <ul css={BylineCss.bylineList} role="list">
-        {authorImage && (
-          <li
-            css={[
-              BylineCss.ImageWrapper,
-              isRtl ? BylineCss.imageRtl : BylineCss.imageLtr,
-            ]}
-          >
-            <Image
-              css={BylineCss.imageSrc}
-              src={authorImage}
-              alt=""
-              placeholder={false}
-              aspectRatio={[1, 1]}
-            />
-          </li>
-        )}
-        <li>
-          {authorTopicUrl ? (
-            <React.Fragment>
-              <VisuallyHiddenText>{`${authorTranslated}, `}</VisuallyHiddenText>
-              <a
-                css={[BylineCss.link, BylineCss.authorLink]}
-                href={authorTopicUrl}
-                className="focusIndicatorReducedWidth"
-              >
+    const reportingFromTranslated = pathOr(
+      'Reporting from',
+      ['byline', 'reportingFrom'],
+      translations,
+    );
```

Comment:
Could we try to use destructuring here (and optional chaining if necessary) instead of using `pathOr` to get the translations?

```
const {
  byline: { author = 'Author ', articleInformation = 'Article information'} } = translations;
```